java/pom.xml

```diff
@@ -37,6 +37,18 @@
     <tag>HEAD</tag>
   </scm>
   <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.13</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.truth</groupId>
+      <artifactId>truth</artifactId>
+      <version>1.1.3</version>
```

Comment:
Is it OK to pin to a version like this? I know there are attack vectors for pining to version string instead of a commit hash.

Reply:
Maven jars are published and hashed: https://repo1.maven.org/maven2/com/google/truth/truth/1.1.3/ and checksummed when downloaded by Maven.

Reply:
Thanks!